packages/ui/src/working-groups/hooks/useWorkers.ts

```diff
@@ -1,38 +1,39 @@
 import { useMemo } from 'react'
 
 import { asMember } from '@/memberships/types'
+import { WorkerStatus, WorkerStatusTypename } from '@/working-groups/types'
 
-import { useGetWorkersQuery, WorkerFieldsFragment } from '../queries'
+import { useGetWorkersQuery } from '../queries'
 
 interface UseWorkersProps {
   groupId?: string
-  statusIn?: Status[]
+  statusIn?: WorkerStatus[]
 }
 
 export const useWorkers = ({ groupId, statusIn }: UseWorkersProps) => {
-  const options = { variables: { where: { group_eq: groupId } } }
+  const options = {
+    variables: {
+      where: {
+        group_eq: groupId,
+        status_json: statusIn ? { isTypeOf_in: statusIn.map((status) => WorkerStatusTypename[status]) } : undefined,
+      },
+    },
```

Comment:
```suggestion
    variables: {
      where: {
        group_eq: groupId,
        status_json: getStatus(statusIn),
      },
    },
```

and something like this:

```typescript
const getSatus = (statusIn?: WorkerStatus[]) => {
  if (!statusIn) {
    return
  }
  
  return { isTypeOf_in: statusIn.map((status) => WorkerStatusTypename[status]) }
}
```

WDYT? Just to hide the implementation from the `variables` definition. There's so much happening on the `status_json` line and that's not necessary to have that there.

Reply:
The same applies for the other hooks that uses `statusIn`